app/assets/stylesheets/admin/views/_document-collection-group-memberships-index.scss

```diff
@@ -10,3 +10,14 @@
     white-space: nowrap;
   }
 }
+
+.app-view-document-collection-group-memberships-index__heading-links {
+  display: flex;
+  align-items: center;
+
+  & > p:first-child {
+    margin-right: 10px;
+    padding-right: 10px;
+    border-right: 1px solid #b1b4b6;
```

Comment:
... and the Design System [Colour values](https://design-system.service.gov.uk/styles/colour/) here?

Reply:
Done!